engine/admin/volumes/tmpfs.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Using tmpfs mounts
+title: Use tmpfs mounts
+keywords: storage, persistence, data persistence, tmpfs
+---
+
+[Volumes](volumes.md) and [bind mounts](bind-mounts.md) are mounted into the
+container's filesystem by default, and their contents are stored on on the host
+machine.
+
+There may be cases where you do not want to store a container's data on the host
+machine, but you also don't want to write the data into the container's writable
+layer, for performance or security reasons, or if the data relates to
+non-persistent application state.
```

Comment:
I think this example is a bit generic so users might have some trouble understanding the usefulness of tmpfs mounts. The example you have in the index.md page about passwords or sensitive data is so much better, so consider using it here.